backend/src/schema/types/type/Location.gql

```diff
@@ -16,3 +16,12 @@ type Location {
   parent: Location @cypher(statement: "MATCH (this)-[:IS_IN]->(l:Location) RETURN l")
 }
 
+type Location2 {
```

Comment:
We need another name: `LocationHelper` or `LocationFinder`

Reply:
In some months we will not remember what Location2 means

Reply:
> Please find a better name for Location2. Something more meaningful. I gave some examples how to test the variables passed to the mutation.

I did this because there is no option to comment in gql files - it makes zero sense to have two different location objects - especially in a graph database. So I am glad you noticed - I'd like to change it and make one out of it - but I believe its a bit out of scope for this PR. Would be an Issue for it acceptable?

Reply:
As I understand it currently the user object just holds the name of the place he has set in his language - thats a nice touch - so if you set "München" everyone in the world will see "München" instead of Munich - but if we at any time want to display the location on a map that is more then impractical

Reply:
using `LocationMapBox`